src/database.json

```
@@ -1013,6 +1013,12 @@
     "repo": "take-five",
     "desc": "A minimal (no external dependency) REST framework"
   },
+  "talk-lib": {
```

Dashes are not allowed:
```suggestion
  "talk_lib": {
```